omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/resolver.lua.erb

```diff
@@ -43,10 +43,7 @@ upstream_resolver.tarpitify = function(route_id, org_config, internal)
 end
 
 upstream_resolver.acct_erchef_fun = function(route)
-  if route.org_config["couchdb_" .. route.endpoint] == 0 then
-    return "erchef"
-  end
-  return "acct"
+  return "erchef"
```

Comment:
Is there further cleanup to do here to remove the dynamic routing bits for the `acct_erchef` upstream?